code/game/objects/items/stacks/stack_recipe.dm

```diff
@@ -64,6 +140,7 @@
 		W.ini_dir = W.dir
 		W.anchored = FALSE
 		W.state = WINDOW_OUT_OF_FRAME
+	..()
```

Comment:
Does post_build need to call parent here? If so, you should probably add the spacemanDMM directive `SHOULD_CALL_PARENT(TRUE)` to the parent call.

Reply:
Currently it just adds the fingerprint of the builder to the window, but more importantly, it's futureproofing.  If post_build() is ever changed in the future, windows (and the other subtypes of stack_recipe) will likely want to inherit the benefits.  For a similar, but opposite reason, SHOULD_CALL_PARENT(TRUE) should not be used as (while there is none currently) a stack_recipe may want to completely override the proc without calling for parent at all.